package.json

```diff
@@ -10,34 +10,35 @@
     "build": "webpack --mode production"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/preset-react": "^7.8.3",
-    "@geppettoengine/geppetto-client": "file:./geppetto-client",
-    "awesome-typescript-loader": "3.5.0",
-    "babel-loader": "^8.0.6",
-    "css-loader": "^3.4.2",
-    "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^3.2.0",
-    "less-loader": "^5.0.0",
-    "react-svg-loader": "^3.0.3",
-    "style-loader": "^1.1.3",
-    "svg-url-loader": "^4.0.0",
-    "typescript": "2.9.2",
-    "webpack": "^4.41.6",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3"
+    "@geppettoengine/geppetto-client": "file:./geppetto-client"
```

Comment:
Better to not deploy a local dependency on the repo. It's fine for the wip but we have to remember to point to the repo/branch in the feature